### PR TITLE
Nk hw 2

### PR DESCRIPTION
**Pull request by natak8200**

### CODE REVIEW

**Comment by AntBelyi** [comment]
Branch is incorrect. You need to merge into your own branch.
Screenshot_4

File: src/main/java/hw_2/Task1.java [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+        double j= 20.5 / 4;
+        System.out.println(j);
+        // %
+        a = 45;
+        b = 15;
+        c = a % b;
+        System.out.println(c);
+        d = 17 % 3;
+        System.out.println(d);
+        // ++ преф.
```
Comment (AntBelyi) [inline src/main/java/hw_2/Task1.java]: There should be no Latin words

File: src/main/java/hw_2/Task1.java [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+        a = 5;
+        b = ++a;
+        System.out.println(a);
+        System.out.println(b);
+        // ++ пост.
```
Comment (AntBelyi) [inline src/main/java/hw_2/Task1.java]: There should be no Latin words

File: src/main/java/hw_2/Task1.java [hunk 3]
```diff
@@ -0,0 +1,126 @@
...
+        a = 7;
+        b = a++;
+        System.out.println(a);
+        System.out.println(b);
+        // -- преф.
```
Comment (AntBelyi) [inline src/main/java/hw_2/Task1.java]: There should be no Latin words

File: src/main/java/hw_2/Task1.java [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+        a = 17;
+        b = --a;
+        System.out.println(a);
+        System.out.println(b);
+        // -- пост.
```
Comment (AntBelyi) [inline src/main/java/hw_2/Task1.java]: There should be no Latin words

File: src/main/java/hw_2/Task1.java [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+        // !
+        a = 20;
+        b = 7;
+        boolean k1 = a != b;
+        System.out.println(k1);
+        // < >
+        a = 17;
+        b = 34;
+        boolean k2 = a < b;
+        boolean k3 = a >b;
```
Comment (AntBelyi) [inline src/main/java/hw_2/Task1.java]: should be extra space after >. Fix in all places please

File: src/main/java/hw_2/Task2.java [hunk 6]
```diff
@@ -0,0 +1,10 @@
+package hw_2;
+
+public class Task2 {
+    public static void main(String[] args) {
+        int x = 3;
+        int y = 4;
+        System.out.println(x + y + " " + x+y);
```
Comment (AntBelyi) [inline src/main/java/hw_2/Task2.java]: should be extra space before and after '+'

File: src/main/java/hw_2/Task1.java (hunk 2 above)
Comment (natak8200) [reply src/main/java/hw_2/Task1.java]: fixed

File: src/main/java/hw_2/Task1.java (hunk 3 above)
Comment (natak8200) [reply src/main/java/hw_2/Task1.java]: fixed

File: src/main/java/hw_2/Task1.java (hunk 4 above)
Comment (natak8200) [reply src/main/java/hw_2/Task1.java]: fixed

File: src/main/java/hw_2/Task1.java (hunk 5 above)
Comment (natak8200) [reply src/main/java/hw_2/Task1.java]: fixed

File: src/main/java/hw_2/Task2.java (hunk 6 above)
Comment (natak8200) [reply src/main/java/hw_2/Task2.java]: fixed

**Pull request closed without merging**